### PR TITLE
Update HPC recipe

### PR DESCRIPTION
**Pull request by jpata**
Provide instructions on how to run the jobs on the Caltech HPC.
Also update the coffea version, restrict btag SF precompilation to only float32 for speed, please do `git submodule update`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,54 @@ cd ../..
...
+
+#make a working directory under the shared filesystem
+mkdir /central/groups/smaria/$USER
+cd /central/groups/smaria/$USER
+mkdir hmm
+cd hmm
+
+#get the code, compile the C++ helper library
+git clone https://github.com/jpata/hepaccelerate-cms
+cd hepaccelerate-cms/tests/hmm
```
Comment (irenedutta23) [inline README.md:85]: Perhaps you also want to add the following two lines?
`git submodule init`
`git submodule update`

Comment (nanlu06) [reply README.md:85]: agreed. also

In the README.md line 102: 

python verify_analyze.py => python verify_analyze.py args_analyze.txt

File: README.md [hunk 2]
```diff
@@ -58,6 +58,54 @@ cd ../..
...
+make -j4
+cd ../..
+
+#run a local test on the interactive login node and create jobfiles
+./tests/hmm/run_hpc.sh
+
+#submit batch jobs
+cd batch
+./make_submit_jdl.sh
+mkdir logs
```
Comment (irenedutta23) [inline README.md:95]: The `slurm_*.out` files are actually getting generated in the `batch` directory itself. So either the `logs` directory isn't needed or these file output location should be modified

File: README.md (hunk 1 above)
Comment (jpata) [reply README.md:87]: thanks. it should be `python verify_analyze.py slurm_submit.sh` on HPC.

File: README.md (hunk 2 above)
Comment (jpata) [reply README.md]: yeah, you can configure the job output directory as you prefer using `sbatch -o logs/...`.

File: README.md [hunk 3]
```diff
@@ -58,6 +58,55 @@ cd ../..
...
+git clone https://github.com/jpata/hepaccelerate-cms
+git submodule init
+git submodule update
+cd hepaccelerate-cms/tests/hmm
+make -j4
+cd ../..
+
+#run a local test on the interactive login node and create jobfiles
+./tests/hmm/run_hpc.sh
+
```
Comment (irenedutta23) [inline README.md:93]: Please also add this `export SUBMIT_DIR=$SUBMIT_DIR` to the instructions. Thanks!

**Comment by nanlu06** [comment]
Thank you! @jpata  This is really great.

**Pull request merged**